Documentation/api-reference.md

```diff
@@ -0,0 +1,138 @@
+# Adobe Experience Platform Identity for Edge Network Extension - Android
+
+## Prerequisites
+
+Refer to the [Getting Started Guide](getting-started.md)
+
+## API reference
+
+| APIs                                           |
+| ---------------------------------------------- |
+| [setAdvertisingIdentifier](#setAdvertisingIdentifier) |
+
+------
+
+### setAdvertisingIdentifier
+
+When this API is called with a valid advertising identifier, the Identity for Edge Network extension includes the advertising identifier in the XDM Identity Map using the _GAID_ namespace. If the API is called with the empty string (`""`), `null`, or the nil UUID values, the GAID is removed from the XDM Identity Map (if previously set).
+
```

Comment:
nit: format `nil` like other values.

Reply:
nit: Also let's add reference to UUID do we give the context. On the contrary do we need to set it as acronym or we can say unique identifer.  Also if we have full form for GAID, let's mention it somewhere.

Reply:
I was thinking to use `null` as opposed to `nil` because of the Java/Kotlin syntax; however, in the UUID case I used `nil` because it seems that the standard terminology uses nil: https://en.wikipedia.org/wiki/Universally_unique_identifier#Nil_UUID

What do you think? I can definitely update it to `nil` instead if you feel that it will improve consistency.

That makes sense, I can link/spell out the terms the first time they are used in the document to provide more context

Reply:
I agree we should use 'null' for examples in this repo.

Reply:
If you think it helps you can have a reference what an advertising identifer is. something on these lines:
Advertising Identifer is an UUID, which is obtained using so and so Android API.

Reply:
Sorry I think I'm a bit confused, do you mean removing all-zeros UUID from the list of special cases? Something like this:

If the API is called with the empty string (`""`) or `null`, the GAID is removed from the XDM Identity Map (if previously set). An advertising identifier is a device-level unique ID in the UUID format, obtained using ...